### DISCUSSION TITLE
Compare with Saleor and Vendure

### DISCUSSION THREAD

**Discussion by kawpii** [medusajs/medusa#2229] [Q&A]
Both of the following are competitors in ecommerce space, like to see a feature comparison and how Medusa is different.

- https://github.com/saleor/saleor
- https://github.com/vendure-ecommerce/vendure

**Comment by akamanocha** [comment]
https://www.google.com/url?sa=t&source=web&rct=j&url=https://dev.to/citizengovind/review-saleor-vs-medusa-two-opensource-headless-ecommerce-platforms-2h4b&ved=2ahUKEwjQ5cKN9pz8AhWR0XMBHSOSCGcQFnoECA8QAQ&usg=AOvVaw145BJDq_8dgzmwL4zns8_o 

This probably might help for comparison with saleor at least

**Comment by pevey** [comment]
The biggest difference between Medusa and Saleor is the base programming language.  The core Medusa server is a Node.js (javascript) app.  Saleor is a python app.  If you are more familiar with one or the other of those, that is the one I would choose if I were you.

Vendure is one to watch.  As of now, it is not as far along in its maturity as Medusa.  The releases are also coming at a much slower pace than updates to Medusa.

**Reply by timuric** [reply]
Hey @pevey I would disagree that programming language is a differentiator, Saleor extensibility is done via GraphQL which is language agnostic.

**Comment by SGFGOV** [comment]
If this helps

https://dev.to/citizengovind/review-saleor-vs-medusa-two-opensource-headless-ecommerce-platforms-2h4b

Best Regards
Govind Diwakar

On Sun, 16 Jul, 2023, 15:26 Timur Carpeev, ***@***.***> wrote:

> Hey @pevey <https://github.com/pevey> I would disagree that programming
> language is a differentiator, Saleor extensibility is done via GraphQL
> which is language agnostic.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/medusajs/medusa/discussions/2229#discussioncomment-6459685>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXEQJHCSETDJCKA67WPE7WTXQO3EBANCNFSM6AAAAAAQPMBQLA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 

This message (including any attachments) may contain confidential, 
proprietary, privileged and/or private
information. The information is 
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the 
sender immediately, and delete the
message and any attachments. Any 
disclosure, reproduction, distribution or other use of this message or
any 
attachments by an individual or entity other than the intended recipient is 
prohibited.

**Comment by kawpii** [comment]
Here is a good comparison table : https://headlesscommerceplatforms.com/, but somethings may be out of date since repo was last updated on Feb 2022 and repo is owned by a Vendure Commerce co-founder.
